lib/utils.js

```diff
@@ -167,11 +167,15 @@ async function simExists (udid) {
   }).reduce((a, b) => {
     return a.concat(b);
```

Comment:
these returns are redundant

Reply:
reduce can be replaced with flatMap

Reply:
I'd rather not have to mess with polyfills to get functionality we can already get from existing functions.

Reply:
I mean lodash's flatMap